packages/vow/src/vow-utils.js

```diff
@@ -73,3 +74,25 @@ export const toPassableCap = k => {
   return vowV0;
 };
 harden(toPassableCap);
+
+/** @param {MakeVowKit} makeVowKit */
+export const makeAsVow = makeVowKit => {
+  /**
+   * Helper function that coerces the result of a function to a Vow. Helpful
+   * for scenarios like a synchronously thrown error.
+   * @template {any} T
+   * @param {(...args: any[]) => Vow<Awaited<T>> | Awaited<T>} fn
+   * @returns {Vow<Awaited<T>>}
+   */
+  const asVow = fn => {
+    const kit = makeVowKit();
+    try {
+      kit.resolver.resolve(fn());
+    } catch (e) {
+      kit.resolver.reject(e);
+    }
+    return kit.vow;
```

Comment:
I think we could simply support not resolving a vow to a vow with the following:

```suggestion
    let result;
    try {
      result = fn();
    } catch (e) {
      result = Promise.reject(e);
    }
    if (isVow(result)) {
      return result;
    }
    const kit = makeVowKit();
    kit.resolver.resolve(result);
    return kit.vow;
```